src/main/java/com/clussmanproductions/trafficcontrol/item/ItemTrafficLight6Frame.java

```diff
@@ -0,0 +1,75 @@
+package com.clussmanproductions.trafficcontrol.item;
+
+import java.util.List;
+
+import com.clussmanproductions.trafficcontrol.ModBlocks;
+import com.clussmanproductions.trafficcontrol.blocks.BlockBaseTrafficLight;
+import com.clussmanproductions.trafficcontrol.blocks.BlockTrafficLight5Upper;
+import com.clussmanproductions.trafficcontrol.gui.GuiProxy;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.items.CapabilityItemHandler;
+import net.minecraftforge.items.IItemHandler;
+
+public class ItemTrafficLight6Frame extends BaseItemTrafficLightFrame
+{
+	public ItemTrafficLight6Frame()
+	{
+		super("traffic_light_6_frame");
+	}
+	
+	@Override
+	public int getBulbCount()
+	{
+		return 4;
+	}
+	
+	@Override
+	public void addInformation(ItemStack stack, World worldIn, List<String> tooltip, ITooltipFlag flagIn) 
+	{
+		IItemHandler handler = stack.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, null);
+		ItemStack subStack = handler.getStackInSlot(0);
+		if (subStack != ItemStack.EMPTY)
+		{
+			tooltip.add("Top: " + subStack.getItem().getItemStackDisplayName(subStack));
+		}
+		
+		subStack = handler.getStackInSlot(1);
+		if (subStack != ItemStack.EMPTY)
+		{
+			tooltip.add("Upper Middle: " + subStack.getItem().getItemStackDisplayName(subStack));
```

Comment:
Top/Upper Middle/Lower Middle/Bottom doesn't really make sense in the context of a T-Frame. Consider naming these more accurately